src/fixes/leading_character.rs

```diff
@@ -0,0 +1,266 @@
+use super::Fix;
+use crate::common::*;
+
+pub(crate) struct LeadingCharacterFixer<'a> {
+    name: &'a str,
+}
+
+impl Default for LeadingCharacterFixer<'_> {
+    fn default() -> Self {
+        Self {
+            name: "LeadingCharacter",
+        }
+    }
+}
+
+impl Fix for LeadingCharacterFixer<'_> {
+    fn name(&self) -> &str {
+        self.name
+    }
+
+    fn fix_line(&self, line: &mut LineEntry) -> Option<()> {
+        let key = line.get_key()?;
+
+        let cleaned_key = remove_invalid_leading_chars(&key);
+
+        line.raw_string = format!("{}={}", cleaned_key, line.get_value()?);
+
+        Some(())
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::path::PathBuf;
+
+    #[test]
+    fn fix_leading_period() {
```

Comment:
What does `period` mean in that case?

Reply:
it just means to fix an item with a leading "." ... I have other tests for some other specific characters like a leading space, asterisk/*, etc. I can remove this test or rename if it helps to make it clearer?

Reply:
Maybe this test should be named `fix_leading_dot`?

Reply:
no problem, just renamed it to that! thank you